core/src/i18n/en.yaml

```
@@ -189,11 +189,15 @@ clusters:
   buttons:
     apply-configuration: Apply configuration
     apply-kubeconfig: Apply kubeconfig
+    verify-and-add: Verify & add
```

```suggestion
    verify-and-add: Verify and Add
```
Formally, we need [title case for buttons](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/79ce282052d14cc38e6a203a1fc277c4.html?q=buttons). Also we can't use [ampersands](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/0511266eabeb4067b395b55e41280f26.html?q=ampersand) here.

Contentwise, do we really need to mention `verify` on the button or is this just a sub-step in order to `add`?  Meaning, why not simply call the button `Add`?

---

To be honest, I was simply using what was given on designs. Just "Add" is fine with me